webcompat/static/css/src/issue-wizard.css

```diff
@@ -225,7 +225,7 @@
   opacity: 0;
 }
 
-.step-container.step10 {
+.step-container.step9 {
```

Comment:
This reminds me of https://github.com/webcompat/webcompat.com/issues/3106 -- should it be closed?

Reply:
yeah, I'll close it. It's actually in the project :)